schemas/objects/sections/homePage/homePageTopSection.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  type: 'object',
+  name: 'homePageTopSection',
+  title: 'Top Banner Section',
+  fields: [
+    {
+      title: 'Title',
+      name: 'title',
+      type: 'string',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Body',
+      name: 'body',
```

Comment:
just a side note, seems like sometimes we use 'description' and some other times 'body', not something to tackle here but maybe good to start thinking about having more consistent naming conventions...